src/openms/include/OpenMS/QC/FragmentMassError.h

```diff
@@ -90,6 +92,27 @@ namespace OpenMS
      */
     void compute(FeatureMap& fmap, const MSExperiment& exp, const QCBase::SpectraMap& map_to_spectrum, ToleranceUnit tolerance_unit = ToleranceUnit::AUTO, double tolerance = 20);
 
+    /**
+      * @brief computes FragmentMassError in ppm and Dalton (only of the first PeptideHit of each PepID)
+      *
+      * Stores average and variance of FragmentMassErrors in ppm as a struct in a vector.
+      * Each FragmentMassError (in ppm) is stored in the first PeptideHit of the corresponding PeptideIdentification as metavalue "fragment_mass_error_ppm".
```

Comment:
The documentation could be improved as it leaves a lot of room for interpretation what is calculated and how it is stored.

E.g. is the only the average and variance stored? Average over what? All peptide hits of a spectrum? "as a struct in a vector" needs to be more precise. Is the FragmentMassError (in ppm) only calculated for the first PeptideHit or the average over all or both? etc.